packages/xo-server-sdn-controller/src/ovsdb-client.js

```diff
@@ -167,7 +181,12 @@ export class OvsdbClient {
     return bridgeName
   }
 
-  async resetForNetwork(networkUuid, networkName) {
+  async resetForNetwork(
+    networkUuid,
+    networkName,
+    crossPoolOnly,
```

Comment:
Can it be both?

Reply:
It's either `crossPoolOnly` then the method only removes crossPool conenctions.
If not it removes all connections.